.github/workflows/python-app.yml

```diff
@@ -0,0 +1,33 @@
+name: Pycee2
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
Do we have any conflicts with 3.8?

Reply:
The latest version we can use is 3.7. That's because PythonBuddy is also using 3.7 and a seamless integration would occur if we're using the same version as they are. We can decide between. I assumed we could use 3.6 as a personal choice but we could also use 3.7 as well.

Reply:
Since PythonBuddy is using 3.7, I believe we can use it too.